src/adapters/integration.test.ts

```diff
@@ -12,9 +13,9 @@ import { Protocol } from './protocols'
 import { testCases as stargateTestCases } from './stargate/tests/testCases'
 import { testCases as uniswapV3TestCases } from './uniswap-v3/tests/testCases'
 
-const TEST_TIMEOUT = 10000
+const TEST_TIMEOUT = 30000
```

Comment:
Ok for now as I've seen timeout errors too, but if we keep seeing errors in the pipeline, we should add some sort of retry mechanism for the tests that fail.